mmdet3d/core/bbox/structures/coord_3d_mode.py

```diff
@@ -243,6 +243,9 @@ def convert_point(point, src, dst, rt_mat=None):
                 f'Conversion from Coord3DMode {src} to {dst} '
                 'is not supported yet')
 
+        if isinstance(rt_mat, np.ndarray):
```

Comment:
Can we modify the if condition to be consistent with [here](https://github.com/open-mmlab/mmdetection3d/blob/a1e04538295b02d42fc59bb7ed58c59c98220174/mmdet3d/core/bbox/structures/coord_3d_mode.py#L146)? After all, they are in the same file. It would be better to have a consistent style.

Reply:
Done!